src/style.css

```diff
@@ -1,3 +1,75 @@
+:root {
+  --link-color: #000;
+  --link-active-color: #00acdc;
+}
+
+html {
+  font-size: 10px;
+  font-family: Roboto, sans-serif;
+}
+
 body {
   margin: 0;
+  padding: 0;
+  box-sizing: border-box;
```

Comment:
The `box-sizing: border-box;` should be applied globally to all elements for consistent sizing. Consider moving it to the `html` selector or use the `*` selector for this specific case, despite the general rule of avoiding the `*` selector.